packages/query/src/graphql/plugins/GetMetadataPlugin.ts

```diff
@@ -1,48 +1,45 @@
 // Copyright 2020-2021 OnFinality Limited authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import * as url from 'url';
 import {makeExtendSchemaPlugin, gql} from 'graphile-utils';
 import fetch from 'node-fetch';
+// @ts-ignore
+import {version} from '../../../package.json';
 import {setAsyncInterval} from '../../utils/asyncInterval';
 import {argv} from '../../yargs';
 
-const metaUrl = argv('meta') as string | undefined;
+const indexerUrl = argv('indexer') as string | undefined;
 
 type MetaData = {
-  currentProcessingHeight: number;
-  currentProcessingTimestamp: number;
   lastProcessedHeight: number;
   lastProcessedTimestamp: number;
   targetHeight: number;
-  uptime: number;
-  polkadotSdkVersion: string;
-  apiConnected: boolean;
-  injectedApiConnected: boolean;
   chain: string;
   specName: string;
   genesisHash: string;
-  blockTime: number;
+  indexerHealthy: boolean;
+  indexerNodeVersion: string;
+  queryNodeVersion: string;
 };
 
-let metaCache: MetaData;
+const metaCache = {
+  queryNodeVersion: version,
+} as MetaData;
 
 export const GetMetadataPlugin = makeExtendSchemaPlugin((build) => {
   return {
     typeDefs: gql`
       type Metadata {
```

Comment:
use `_Metadata` to differentiate it from user specified entities

Reply:
Fixed